### ISSUE TITLE
Accessibility: contrast ratio gov-notification

### ISSUE THREAD

**Issue by K-T-O** [CJSCommonPlatform/govuk_single_page_pdk#101]
Pa11y files an error about the following issue:

- WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail

Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 1.81:1. Recommendation: change background to #939597.

This occurs when the following CSS classes are used together: gov-notification inactive.

**Comment by jamesbirrellgray** [comment]
@K-T-O - I can't see where gov-notification is used with inactive in the PDK docs... http://cjscommonplatform.github.io/govuk_single_page_pdk/#/components/banners

**Comment by fabiotisci** [comment]
gov-notification inactive was removed as the new CJS CPP Document didn't contemplate that background color. Please update ... the most similar one would be 'deregistered'. Previous one used $grey-2 as background color, the new one uses $grey-1

**Comment by blackmonday** [comment]
Can a visual/screengrab of the component in question be provided as it's not clear to me what is being discussed.

**Comment by jamesbirrellgray** [comment]
@blackmonday - got you :-) - docs are here http://cjscommonplatform.github.io/govuk_single_page_pdk/#/components/banners

Screenshot attached

<img width="1021" alt="screen shot 2016-10-26 at 11 07 20" src="https://cloud.githubusercontent.com/assets/16034882/19721975/7cd38e22-9b6c-11e6-8ad4-c2c9cb8fff94.png">

**Comment by blackmonday** [comment]
Is it the grey version that this issue is about?

The grey version contrast ratio is 4.56:1 and passes AA

**Comment by jamesbirrellgray** [comment]
@blackmonday @K-T-O : It looks like @K-T-O is using a class not specified in his code base - The issue has been closed as this is not a PDK issue but an implementation issue.

**Comment by blackmonday** [comment]
Thanks, James.

**Comment by K-T-O** [comment]
@jamesbirrellgray This is not a problem with implementation but with earlier versions of PDK (https://github.com/CJSCommonPlatform/govuk_single_page_pdk/blob/0.0.100/src/modules/components/banners/notification/notification.scss) which is not in use any more. Please note that this issue was filed on Sep 15th when an older version of PDK was used by IDAM.

**Comment by jamesbirrellgray** [comment]
@K-T-O This is not a blame game (we don't have that culture here) :-), there have been changes in design that have flowed to PDK in updated versions, your code now contains a deprecated implementation (due to design and PDK changes), so therefore it is implementation (all be its as a result of PDK changes).
